resources/integration_test/tests/query.yaml

```diff
@@ -0,0 +1,197 @@
+    expected_column_label: [ e, n, s ]
+    expected_column_display_size: [ 1, 0, 10 ]
+    expected_precision: [ 1, 0, 10 ]
+    expected_scale: [ 0, 0, 0 ]
+    expected_schema_name: [ '', '', '' ]
+    expected_is_auto_increment: [ false, false, false ]
+    expected_is_case_sensitive: [ false, true, false ]
+    expected_is_currency: [ false, false, false ]
+    expected_is_definitely_writable: [ false, false, false ]
+    expected_is_nullable: [ columnNullable, columnNullable, columnNullable ]
+    expected_is_read_only: [ true, true, true ]
+    expected_is_searchable: [ true, true, true ]
+    expected_is_signed: [ false, false, true ]
+    expected_is_writable: [ false, false, false ]
+
+  - description: inner_join
+    db: integration_test
+    sql: "SELECT class.name, grades.testid, grades.score FROM grades INNER JOIN class ON grades.studentid\
+      = class.studentid"
+    ordered: false
+    skip_reason: SQL-659
+
+  - description: subquery
+    db: integration_test
+    sql: SELECT studentid FROM (SELECT * FROM class WHERE enrolled = TRUE) AS foo
+    ordered: false
+    skip_reason: SQL-659
+
+  - description: count_and_group_by
+    db: integration_test
+    sql: SELECT COUNT(score), testid FROM grades WHERE score > 85 GROUP BY testid
+    ordered: false
+    skip_reason: SQL-658
+
+  - description: driver_does_not_alter_result_set_row_order
+    db: integration_test
+    sql: SELECT grades.score AS score FROM grades ORDER BY score DESC
+    ordered: true
+    skip_reason: SQL-659
+
+  - description: select_star_from_anyof
+    db: integration_test
+    skip_reason: SQL-668
+
+  - description: select_star_from_any
+    db: integration_test
+    sql: SELECT * from any
+    skip_reason: SQL-696
+
+  - description: select_field_from_any
+    db: integration_test
+    sql: SELECT b from any_collection
+    expected_result:
+      - - !!org.bson.BsonInt32  3
+      - - !!org.bson.BsonString  b
+      - - !!org.bson.BsonDouble  4.5
+    row_count: 3
+    row_count_gte: null
+    ordered: null
+    expected_sql_type: [ OTHER ]
+    expected_catalog_name: [ '' ]
+    expected_column_class_name: [ org.bson.BsonValue ]
+    expected_column_label: [ b ]
+    expected_column_display_size: [ 0 ]
+    expected_precision: [ 0 ]
+    expected_scale: [ 0 ]
+    expected_schema_name: [ '' ]
+    expected_is_auto_increment: [ false ]
+    expected_is_case_sensitive: [ true ]
+    expected_is_currency: [ false ]
+    expected_is_definitely_writable: [ false ]
+    expected_is_nullable: [ columnNullable ]
+    expected_is_read_only: [ true ]
+    expected_is_searchable: [ true ]
+    expected_is_signed: [ true ]
+    expected_is_writable: [ false ]
+
+  - description: nonexist_field_return_null
+    db: integration_test
+    sql: SELECT nonexist FROM class
+    skip_reason: SQL-698
+
+  - description: select_null_field
+    db: integration_test
+    sql: SELECT a FROM null_field
+    expected_result:
+      - [a]
+      - [null]
```

Comment:
We could consider combining these into one test:
```
# data.yml
...
    - db: db
      collection: null_and_missing
      data:
          - { _id: 0, a: "a" }
          - { _id: 1, a: null }
          - { _id: 2 }
...

# tests/query.yml
...
    - description: select_null_and_missing_field
      sql: SELECT a FROM null_and_missing
      expected_result:
        - [a]
        - [null]
        - [null]
...
```
This test would need the SQL-698 skip-reason then, but that's ok

Reply:
Sounds good, combined the tests.